README.md

```diff
@@ -62,11 +62,11 @@ docsify init [path] [--local false] [--theme vue] [--plugins false]
   - Type: boolean
   - Default: `false`
   - Description: Copy `docsify` files to the docs path, defaults to `false` using `cdn.jsdelivr.net` as the content delivery network (CDN). To explicitly set this option to `false` use `--no-local`.
-- `--theme` option:
-  - Shorthand: `-t`
-  - Type: string
-  - Default: `vue`
-  - Description: Choose a theme, defaults to `vue`, other choices are `buble`, `dark` and `pure`.
+- `--previewRelease` option:
```

Comment:
The option name is too long.

Reply:
I think replacing the version number and adding HTML content separately are the same thing.

Generate different HTML content based on different versions and perform replacements.

`--local` compatible with scenarios using local docsify.

Reply:
> I think replacing the version number and adding HTML content separately are the same thing.
for instance now.

If we separated the templates/resources by versions thats is feasible. 
But the commands are incompatible and hard to handle.
i.e. The `theme` is only support `vue` now, If user wanna use v4 docsify, how does he use the `--themes` of it?